chat-widget/src/common/utils/xssUtils.ts

```diff
@@ -0,0 +1,34 @@
+import DOMPurify from "dompurify";
+
+const xssPatterns = [
```

Comment:
this would not be a complete list of patterns, right? why cannot we always use the sanitized title?

Reply:
if the text is flagged as XSS, it will be a risk to use the text, is better to default to predefined text